src/interface/others/RestorationShamanSpreadsheet.js

```diff
@@ -60,7 +59,6 @@ class RestorationShamanSpreadsheet extends React.Component {
               <tr><td>{parser.getModule(SurgingTides).surgingTideProcsPerMinute}</td></tr>
               <tr><td>{((parser.getModule(SpoutingSpirits).spoutingSpiritsHits || getAbility(SPELLS.SPOUTING_SPIRITS_HEAL.id).healingHits) / casts(SPELLS.SPIRIT_LINK_TOTEM.id) || 0).toFixed(2)}</td></tr>
               <tr><td>{((parser.getModule(OverflowingShores).overflowingShoresHits || getAbility(SPELLS.OVERFLOWING_SHORES_HEAL.id).healingHits) / casts(SPELLS.HEALING_RAIN_CAST.id) || 0).toFixed(2)}</td></tr>
-              <tr><td>{parser.getModule(EbbAndFlow).ebbAndFlowEffectiveness.toFixed(2)}</td></tr>
```

Comment:
please return a `0` instead of removing the line as the order is important for the spreadsheet which uses this